conf/gunicorn_verbs.py

```diff
@@ -1,3 +1,8 @@
+# NOTE: Must be before we import or call anything that may be synchronous.
+from gevent import monkey
```

Comment:
We don't use gevent in verbs, so this isn't needed

Reply:
@josephschorr In that case, would you be opposed to me removing the `gunicorn_verbs.py` file and pointing supervisord to start verbs as a standard python module?

Current:
```
command=nice -n 10 gunicorn -c %(ENV_QUAYCONF)s/gunicorn_verbs.py verbs:application
```

Proposed (untested)
```
python -m verbs:application
```

Reply:
No. It still uses *gunicorn*, just not *gevent*. It uses sync workers.

Reply:
Thanks! Closing this accordingly.